src/Microsoft.Sbom.Api/Hashing/IHashAlgorithmProvider.cs

```diff
@@ -7,5 +7,5 @@ namespace Microsoft.Sbom.Api.Hashing;
 
 public interface IHashAlgorithmProvider
 {
-    AlgorithmName Get(string algorithmName);
+    public AlgorithmName Get(string algorithmName);
```

Comment:
Are all of these 'public' changes necessary? I would be fine with these changes but I would prefer that they be split off into a separate PR if possible

Reply:
I can pull them out - heads-up that any 9.0.200 usage is going to bring them back though